pyat/at/load_mat.py

```diff
@@ -100,52 +102,33 @@ def find_class_name(kwargs):
                 elif hasattrs(kwargs, 'K'):
                     return 'Quadrupole'
                 elif hasattrs(kwargs, 'PolynomB'):
-                    try:
-                        PolynomB = numpy.array(kwargs['PolynomB'],
-                                               dtype=numpy.float64)
-                        loworder = numpy.where(PolynomB != 0.0)[0][0]
-                    except IndexError:
-                        if kwargs.get('PassMethod') == 'StrMPoleSymplectic4Pass':
-                            return 'Multipole'
-                        elif hasattrs(kwargs, 'BendingAngle'):
-                            if float(kwargs['BendingAngle']) == 0.0:
-                                return 'Drift'
-                            else:
-                                return 'Bend'
-                        else:
-                            return 'Drift'
+                    loworder = low_order('PolynomB')
+                    # loworder = min(low_order('PolynomB'), low_order('PolynomA'))
```

Comment:
Delete this line? I don't see how it would be useful even if included?

Reply:
I prepared this line for the case you would insist in considering skew quads as quadrupoles (that's also why I set `low_order` as a function. If we can keep skew elements as `Multipole`s, I remove it.